### ISSUE TITLE
Completion submission completion status

### ISSUE THREAD

**Issue by Janetbananet** [responsible-ai-collaborative/aiid#1786]
We need a way to quickly determine the completeness of a submission in the review queue. This may be judged by the number of completed fields, but that needs more thought/discussion.

The "Quick Add URL's" should automatically be designated a 10% completion, and then be combined with the rest of the submissions so that there is only one central queue.

**Comment by Janetbananet** [comment]
Whoever picks this one up needs to work with @LKchemposer since he will be the primary user for this feature at first.

**Comment by smcgregor** [comment]
Our goal is to empty the queue and the only exceptions are when something needs editor discussion or an advancement in the software before we can ingest it. The "completion status" presentation of the form would help to prioritize between the quick adds and the full submissions when they are sitting in the same queue. Maybe we handle the "editor hold" submissions with better presentation of the editor's notes and handle the quick submissions in the queue with a simple counter of the number of filled form fields. Thoughts @LKchemposer?

**Comment by Janetbananet** [comment]
@clari182 Can we make sure this GitHub issue stays open after we close #2070 since we took the completion status out of scope?

**Comment by clari182** [comment]
Hey @Janetbananet , yes, this issue can remain open.
